addons/website/static/src/snippets/s_card/001.scss

```diff
@@ -0,0 +1,41 @@
+.s_card {
+    --card-img-size-h: 30%;
+    --card-img-ratio-align: 44%;
+    --card-img-aspect-ratio: 20%;
+
+    min-width: MIN(18rem, 100%);
+    margin: $grid-gutter-width/2 0;
+
+    .o_card_img {
+        width: 100%;
+    }
+
+    // When a color-preset is applied, let 'card-body' inherit its
+    // background and text colors.
+    &.o_cc .card-body {
+        background-color: inherit;
+        color: inherit;
+    }
```

Comment:
Thanks, great analysis 💯 

> either like I said with `.card:not([data-vcss]) .card-body`, or by finding a way to somehow overwrite it

I opted for `:where(.card:not([data-vcss])) .card-body` that has a [lower specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/:where).  
It solves any colors related issue for the new cards 👍 

---

> the background should only be applied on the card-body

It's not doable because Bootstrap cards could contain different type of content, like `.card-header`, `.card-footer`, `.list-group` and **multiple** `.card-body` 😅 ([ref](https://getbootstrap.com/docs/5.3/components/card/#kitchen-sink).)Here what you would get eventually  ⬇︎

![image](https://github.com/user-attachments/assets/245ae4e8-74ac-413f-a9a2-52a65025b4ee)

...**but** there is no problem to solve IMO 🙂 To have **both** a cover image and a background image  it's a very corner case scenario... the only circumstance I can think of  it's when the background it's used a decoration, the same for shapes 😉  

![image](https://github.com/user-attachments/assets/cc4bb500-41b5-4ed2-9877-ed45dcab2bb2)